qemu/tests/cfg/hotplug_mem_share_discard_data.cfg

```diff
@@ -0,0 +1,38 @@
+- hotplug_mem_share_discard_data:
+    only Linux
+    type = hotplug_mem_share_discard_data
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.10
```

Comment:
No platform limitation for this case, be careful with other platform's threshold, such as ppc: `threshold = 0.15`

Reply:
@yanan-fu 

Reviewing the case again I realized that I am not using the threshold (that's because I use other memory hotplug cases as a template :upside_down_face: ) so I think deleting it it's fine, do you agree?